src/main/java/net/earthcomputer/clientcommands/c2c/C2CPacket.java

```diff
@@ -1,9 +1,10 @@
 package net.earthcomputer.clientcommands.c2c;
 
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
 import net.minecraft.network.PacketByteBuf;
 
 public interface C2CPacket {
     void write(PacketByteBuf buf);
 
-    void apply(CCPacketListener listener);
+    void apply(CCPacketListener listener) throws CommandSyntaxException;
```

Comment:
I feel like this is out of the scope of command syntax exceptions. Perhaps we should create our own exception class or just `return` instead of throwing an exception.